backend/cognito/cognito.template.yml

```diff
@@ -414,13 +460,14 @@ Resources:
         SnsCallerArn: !GetAtt SmsRole.Arn
         ExternalId: !Ref ExternalId
         SnsRegion: !Ref AWS::Region
+      LambdaConfig:
+        CustomSMSSender:
+          LambdaArn: !GetAtt SecurityCodeTextMessageSender.Arn
+          LambdaVersion: V1_0
```

Comment:
```suggestio
```
This is not required

Reply:
In cognito documentation it says it is required:
[cognito doc](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_CustomSMSLambdaVersionConfigType.html#CognitoUserPools-Type-CustomSMSLambdaVersionConfigType-LambdaVersion)

During deployment I get error:
`Value null at 'lambdaConfig.customSMSSender.lambdaVersion' failed   to satisfy constraint: Member must not be null
(Service: AWSCognitoIdentityProviderService; Status Code: 400; Error Code: InvalidParameterException; Request ID: 8cfd74be-e070-4d2a-8733-b9931b1da645; Proxy: null)
`

Reply:
In [cognito documentation](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_CustomSMSLambdaVersionConfigType.html#CognitoUserPools-Type-CustomSMSLambdaVersionConfigType-LambdaVersion) it says it is required.

During deployment I get error:
`Value null at 'lambdaConfig.customSMSSender.lambdaVersion' failed   to satisfy constraint: Member must not be null
(Service: AWSCognitoIdentityProviderService; Status Code: 400; Error Code: InvalidParameterException; Request ID: 8cfd74be-e070-4d2a-8733-b9931b1da645; Proxy: null)
`

Reply:
I check CloudFormation documentation for [CustomSMSSender](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cognito-userpool-customsmssender.html) and it says it's not required 🤔 
Thanks for checking, though, we'll leave it in then